src/reactRouter.re

```diff
@@ -35,10 +35,52 @@ module Route = {
     );
 };
 
+module Switch = {
+  [@bs.module "react-router-dom"] external _switch : ReasonReact.reactClass = "Switch";
+  let make = (children) =>
+    ReasonReact.wrapJsForReason(
+      ~reactClass=_switch,
+      ~props=Js.Obj.empty(),
+      children
+    );
+};
+
+module Link = {
+  [@bs.module "react-router-dom"] external link : ReasonReact.reactClass = "Link";
+  let make =
+      (
+        ~_to: string,
```

Comment:
Should we use `_to` or `to_`? Do you know if there is a convention for such cases?

Reply:
That was a good question ! My guess is `to_` because `_to` looks like a non-used variable so I change uses.